backend/events/tests/test_events_list.py

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+import pytest
+from django.test import Client
+
+pytestmark = pytest.mark.django_db
+
+
+def test_event_list(client: Client) -> None:
+    """
```

Comment:
Include the following:

- Number of pages
- Number of events

This can be seen in the API view tests and in organizations.